public/css/user/home.css

```diff
@@ -3,6 +3,266 @@
   padding: 0;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+section {
+  height: 100vh;
+}
+
+.main--header {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.main--header-title {
+  margin-top: 150px;
+  width: 100%;
+  display: flex;
+}
+
+.main--header-titleWhite {
+  font-weight: 300;
+  font-family: Roboto;
```

Comment:
@aminalakhsham If everything on this page should be Roboto, weight 300, you could lift these style attributes up a level or two (put them on `.main--header` or even `body`) and then you won't have to repeat them further down in this file.

Reply:
not every thing is Roboto , there is another font is lato 
but as you siad can but it in body and specific font family lato can stay in there please 
ty :)

Reply:
Ah got it.  That sounds good too.  Whatever will help you from having to write `font-family: roboto` all over the place :)